Makefile

```diff
@@ -80,7 +80,6 @@ server/openapi: $(OPENAPI_SRC_FILES)
 	$(OPENAPI_GENERATOR_CLI) generate \
 		-i api/openapi.yaml \
 		-g go-server \
-		--global-property apis="api" \
```

Comment:
@danielbdias I remember you added this to avoid having the controller.go requiring us to add the missing route controllers right?

Reply:
this is the cause for the server generation not working. Removing this doesn't break the server